src/components/Layout/PhoneNumberInput.tsx

```diff
@@ -71,14 +99,14 @@ export const PhoneNumberInput: FunctionComponent<{
           style={styles.countryCode}
           keyboardType="phone-pad"
           value={countryCodeValue}
-          onChange={({ nativeEvent: { text } }) => onChangeCountryCode(text)}
+          onChangeText={text => onChangeCountryCode(text)}
         />
         <AppText style={styles.hyphen}>-</AppText>
         <TextInput
           style={styles.numberInput}
           keyboardType="phone-pad"
-          value={mobileNumberValue}
-          onChange={({ nativeEvent: { text } }) => onChangeMobileNumber(text)}
+          value={value}
+          onChangeText={text => setValue(formatPhoneNumber(text))}
```

Comment:
Change this back to `onHandleChangePhoneNumber(text)`, I don't think we should rely on `useEffect` for this, sorry!

Reply:
Thanks a lot Shermaine!